net/openvswitch/Makefile

```diff
@@ -134,8 +134,13 @@ define KernelPackage/openvswitch
   KCONFIG:=CONFIG_BRIDGE
   DEPENDS:=+kmod-stp @IPV6 +kmod-gre +kmod-lib-crc32c +kmod-vxlan +kmod-nf-conntrack +kmod-nf-conntrack6 @($(SUPPORTED_KERNELS))
   FILES:= \
+	$(PKG_BUILD_DIR)/datapath/linux/vport-geneve.$(LINUX_KMOD_SUFFIX) \
+	$(PKG_BUILD_DIR)/datapath/linux/vport-gre.$(LINUX_KMOD_SUFFIX) \
+	$(PKG_BUILD_DIR)/datapath/linux/vport-lisp.$(LINUX_KMOD_SUFFIX) \
+	$(PKG_BUILD_DIR)/datapath/linux/vport-stt.$(LINUX_KMOD_SUFFIX) \
+	$(PKG_BUILD_DIR)/datapath/linux/vport-vxlan.$(LINUX_KMOD_SUFFIX) \
 	$(PKG_BUILD_DIR)/datapath/linux/openvswitch.$(LINUX_KMOD_SUFFIX)
-  AUTOLOAD:=$(call AutoLoad,21,openvswitch)
+  AUTOLOAD:=$(call AutoLoad,21,openvswitch vport-geneve vport-gre vport-lisp vport-stt vport-vxlan)
```

Comment:
The version in master branch is now 2.8 and switches to using kernel datapath from vanilla kernel instead of compiling the one from ovs release.  Do you depend on stt and lisp tunnel support?